frontend/src/app/choreography/choreography.component.ts

```diff
@@ -62,19 +67,15 @@ export class ChoreographyComponent {
     }
   }
 
-  get people(): Person[] {
+  get peopleInDatabase(): Person[] {
```

Comment:
**suggestion:** "database" sounds a bit odd in the frontend context so I'd rather say something like "allAvailablePeople" or something